assets/src/edit-story/components/fontPicker/fontList/index.js

```diff
@@ -0,0 +1,269 @@
+    () =>
+      new window.IntersectionObserver(
+        (entries) => {
+          entries.forEach((entry) => {
+            if (entry.isIntersecting) {
+              setUserSeenFonts(addUniqueEntry(entry.target.dataset.font));
+            }
+          });
+        },
+        {
+          root: listRef.current,
+          // Gets ~2 extra items below scroll container BoundingBox
+          rootMargin: '60px',
+        }
+      ),
+    []
+  );
+
+  // Observe rendered font options
+  useLayoutEffect(() => {
+    const renderedOptions = optionsRef.current;
+    renderedOptions.forEach((option) => option && observer.observe(option));
+    return () => {
+      renderedOptions.forEach((option) => option && observer.unobserve(option));
+      // clear exisiting option references before next update to filteredGroup
+      optionsRef.current = [];
+    };
+  }, [observer, filteredListGroups]);
+
+  // load all seen fonts from google service
+  useEffect(() => {
+    ensureMenuFontsLoaded(
+      userSeenFonts.filter(
+        (fontName) => fontMap[fontName]?.service === 'fonts.google.com'
+      )
+    );
+  }, [ensureMenuFontsLoaded, userSeenFonts, fontMap]);
+
+  /*
+   * KEYBOARD ACCESSIBILITY
+   */
+  const filteredOptions = useMemo(() => getOptions(filteredListGroups), [
+    filteredListGroups,
+  ]);
+
+  const handleKeyPress = useCallback(
+    ({ key }) => {
+      if (key === 'Escape') {
+        onClose();
+      } else if (key === 'Enter') {
+        filteredOptions[focusIndex] &&
+          onSelect(filteredOptions[focusIndex].name);
+      } else if (key === 'ArrowUp') {
+        setFocusIndex((index) => Math.max(0, index - 1));
+      } else if (key === 'ArrowDown') {
+        setFocusIndex((index) =>
+          Math.min(filteredOptions.length - 1, index + 1)
+        );
+      }
+    },
+    [focusIndex, filteredOptions, onClose, onSelect]
+  );
+
+  useFocusOut(listRef, () => setFocusIndex(-1), []);
+
+  // scroll into view and focus focusIndex
+  useEffect(() => {
+    const listEl = listRef.current;
+    if (!listEl) {
+      return;
+    }
+    if (focusIndex === -1) {
+      listEl.scrollTo(0, 0);
+      return;
+    }
+
+    const highlighedOptionEl = optionsRef.current[focusIndex];
+    if (!highlighedOptionEl) {
+      return;
+    }
+
+    highlighedOptionEl.focus();
+    listEl.scrollTo(0, highlighedOptionEl.offsetTop - listEl.clientHeight / 2);
+  }, [focusIndex, filteredOptions, keyword]);
+
+  /*
+   * ACCESSIBILITY COMMUNICATION WITH SEARCH INPUT
+   */
+  const isExpanded = filteredOptions.length > 0;
+  useEffect(() => onExpandedChange(isExpanded), [onExpandedChange, isExpanded]);
+  useEffect(() => {
+    if (focusTrigger > 0) {
+      setFocusIndex(0);
+    }
+  }, [focusTrigger]);
+
+  return filteredOptions.length <= 0 ? (
+    <NoResult>{__('No matches found', 'web-stories')}</NoResult>
+  ) : (
+    <List
+      ref={listRef}
+      tabIndex={0}
+      id="editor-font-picker-list"
```

Comment:
Seems a bit dangerous to hardcode (though I can see it was before as well). There could technically be more than one instance of this component.

Reply:
yea good point. I'll see what we're using it for and see if there's another way around hardcoding it

Reply:
so two places it's used here are:
- `FontSearchInput` for an `aria-owns` 
- the Karma tests to select it this element

This isn't bad to fix, but I'll circle back around to it after stable release cus I need to get this PR through today and karma tests are taking forever